wheel.css

```diff
@@ -4,6 +4,12 @@
   fill: #5CB85C;
 }
 
+.warning {
+  stroke: #d58512;
+  stroke-width: 1;
+  fill: #ed9c28;
+}
+
```

Comment:
In light of [this historical comment](https://github.com/meshy/pythonwheels/issues/5#issuecomment-28407992), I wonder if there may be a more broadly accessible colour than orange here.